### ISSUE TITLE
Update to react-spring 7

### ISSUE THREAD

**Issue by swashata** [drcmda/mauerwerk#6]
Hello,

I have tried giving it a go with react-spring 7 and the page kinda froze with some infinite loop (didn't investigate much).

It would be super helpful if you can guide me updating the (awesome, bdw) library to work with latest react-spring.

**Comment by drcmda** [comment]
Yes, would be great to update it. Would you want to try sending in a PR? There's a working version of this over at https://github.com/react-spring/react-spring/tree/master/examples/demos/grid

**Comment by swashata** [comment]
Thank you so very much for the guide. I will try my best. Please give me a couple of days.
